### PR TITLE
VIITE-1439 Remove OTH-specific code part 2

### PR DESCRIPTION
**Pull request by skosonen**

### CODE REVIEW

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/RoadLinkService.scala [hunk 1]
```diff
@@ -93,22 +86,12 @@ class RoadLinkService(val vvhClient: VVHClient, val eventbus: DigiroadEventBus,
 
   def getRoadLinksByLinkIdsFromVVH(linkIds: Set[Long], newTransaction: Boolean = true, frozenTimeVVHAPIServiceEnabled: Boolean = false): Seq[RoadLink] = {
```
Comment (pedrosag) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/RoadLinkService.scala]: Do we need the "newTransaction: Boolean = true," part?

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/RoadLinkService.scala [hunk 2]
```diff
@@ -93,22 +86,12 @@ class RoadLinkService(val vvhClient: VVHClient, val eventbus: DigiroadEventBus,
...
-    if (newTransaction)
-      withDynTransaction {
-        enrichRoadLinksFromVVH(vvhRoadLinks)
-      }
-    else
-      enrichRoadLinksFromVVH(vvhRoadLinks)
+    enrichRoadLinksFromVVH(vvhRoadLinks)
   }
 
   def getSuravageRoadLinksByLinkIdsFromVVH(linkIds: Set[Long], newTransaction: Boolean = true): Seq[RoadLink] = {
```
Comment (pedrosag) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/RoadLinkService.scala]: Do we need the "newTransaction: Boolean = true," part?

Comment (skosonen) [reply digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/RoadLinkService.scala]: I will remove those.

**Comment by skosonen** [comment]
All tests pass on Jenkins: #1104.

**Pull request merged**